environment.py

```diff
@@ -25,19 +26,23 @@ def env() -> Mapping[str, Optional[str]]:
     return {
 
         # Configure the catalogs to be managed by this Azul deployment. A
-        # catalog is a group of indices populated from a particular source. The
-        # syntax is
-        #
-        # catalog, (',', catalog)*
-        #
-        # The syntax of each catalog is
-        #
-        # atlas, ':', catalog_name, (':', plugin_type, '/', plugin_package)+
-        #
-        # where `atlas` is the name of the atlas the catalog belongs to
-        # and `catalog_name` is the name of the catalog. The atlas name and 
-        # catalog names follow the same syntax, which is fairly
-        # restrictive (see azul.IndexName.validate_catalog_name for details).
+        # catalog is a group of indices populated from a particular source.
+        #
+        # The AZUL_CATALOGS variable must be a string that represents a JSON
+        # array of JSON objects with the following fields:
+        #
+        # {
+        #     "name": "foo",
```

Comment:
```suggestion
        #     'name': "foo",
```

and so on

Reply:
Changed but would we not want to use `"` to indicate that its JSON?